### PR TITLE
satellitedb: fixes for cockroachdb compatibility

### PR DESCRIPTION
**Pull request by JessicaGreben**
Why

We are adding support for cockroachDB for the satellitedb. First we need to make sure our tests can handle using cockroachdb.

What: 
This PR does the following:
- updates the storagenodeaccounting query to be postgres and cockroach compatible.  
- fixes naming the databases created for each unit test
- removes unsupported where clause from testdata/postgres

Please describe the tests:
n/a
 
Please describe the performance impact:
n/a

## Code Review Checklist (to be filled out by reviewer)
 - [x] NEW: Are there any Satellite database migrations? Are they forwards _and_ backwards compatible? 
 - [x] Does the PR describe what changes are being made?
 - [x] Does the PR describe why the changes are being made?
 - [x] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [x] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [x] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [x] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [x] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [x] Does any documentation need updating?
 - [x] Do the database access patterns make sense?

### CODE REVIEW

File: satellite/satellitedb/storagenodeaccounting.go [hunk 1]
```diff
@@ -219,25 +219,29 @@ func (db *StoragenodeAccounting) QueryStorageNodeUsage(ctx context.Context, node
...
-			FROM storagenode_storage_tallies
-			WHERE node_id = ?
-			AND NOT EXISTS (SELECT 1 FROM r WHERE DATE(r.start_time) = DATE(interval_end_time))
-			AND (SELECT value FROM accounting_timestamps WHERE name = ?) < interval_end_time AND interval_end_time <= ?
-			GROUP BY start_time
-		ORDER BY start_time`
+		SELECT SUM(data_total) AS at_rest_total, interval_end_time::date AS start_time
+				FROM storagenode_storage_tallies
+				WHERE node_id = $1
+				AND NOT EXISTS (SELECT 1 FROM (
```
Comment (thepaul) [inline satellite/satellitedb/storagenodeaccounting.go]: I think we could combine these two nested selects, now that we've taken apart the CTE anyway, and get

```sql
    AND NOT EXISTS (
        SELECT 1 FROM accounting_rollups
        WHERE node_id = $1
        AND $2 <= start_time AND start_time <= $3
        AND start_time::date = interval_end_time::date
    )
```

File: satellite/satellitedb/testdata/postgres.v70.sql [hunk 2]
```diff
@@ -358,7 +358,7 @@ CREATE UNIQUE INDEX serial_number ON serial_numbers ( serial_number );
 CREATE INDEX serial_numbers_expires_at_index ON serial_numbers ( expires_at );
 CREATE INDEX storagenode_id_interval_start_interval_seconds ON storagenode_bandwidth_rollups ( storagenode_id, interval_start, interval_seconds );
 
-CREATE UNIQUE INDEX credits_earned_user_id_offer_id ON user_credits (id, offer_id) WHERE credits_earned_in_cents=0;
+CREATE UNIQUE INDEX credits_earned_user_id_offer_id ON user_credits (id, offer_id);
```
Comment (thepaul) [inline satellite/satellitedb/testdata/postgres.v70.sql]: Yessss the sooner we get this in, the better, so no one else keeps copying the broken version into new files

Comment (JessicaGreben) [reply satellite/satellitedb/testdata/postgres.v70.sql]: lol! i know, its so annoying!

File: satellite/satellitedb/storagenodeaccounting.go (hunk 1 above)
Comment (JessicaGreben) [reply satellite/satellitedb/storagenodeaccounting.go]: let me make this change and confirm all still looks good

Comment (thepaul) [reply satellite/satellitedb/storagenodeaccounting.go]: @JessicaGreben You are bad at PTO

Comment (JessicaGreben) [reply satellite/satellitedb/storagenodeaccounting.go]: lol I have a free couple of mins.

**Pull request merged**